lib/modem_info/Kconfig

```diff
@@ -6,7 +6,7 @@
 
 menuconfig MODEM_INFO
 	bool "nRF91 modem information library"
-	select NRF_MODEM_LIB
+	depends on NRF_MODEM_LIB
```

Comment:
There are few `imply NRF_MODEM_LIB` also in nrf_provisioning and I'm getting some builds through by changing also those to `depends on NRF_MODEM_LIB`.

Reply:
Thanks, I will try.